azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/service/sdk/ChatSDK.kt

```diff
@@ -17,4 +18,7 @@ internal interface ChatSDK {
         type: ChatMessageType,
         content: String,
     ): CompletableFuture<SendChatMessageResult>
+
+    fun startEventNotifications(context: Context) {}
```

Comment:
We can remove context from here. Wrapper class already have context in constructor.

Reply:
@iaulakh but the context is not being held in a variable so it isn't directly available (outside the constructor), right? I can add an instance variable to hold it and then pass it in, but then we'll probably have to make a de-initialize call to stop holding on to the context otherwise there'll be a memory leak, I think. 🤔

Reply:
sounds good Dhiraj.
internal class ChatSDKWrapper(
    private val context: Context,
    chatConfig: ChatConfiguration,
)

Reply:
Using context as instance variable in https://github.com/Azure/communication-ui-library-android/commit/093c6e208e9d4ce907bb469abfc93e324bc083e8